exporter/cassandraexporter/config.go

```diff
@@ -9,15 +9,15 @@ import (
 )
 
 type Config struct {
+	Auth        Auth          `mapstructure:"auth"`
```

Comment:
Should this be recorded as a breaking change?

Reply:
I suspect it wouldn't be breaking many users, but worth recording in a changelog entry

Reply:
we just moved the field around, there is no API change